### ISSUE TITLE
Misc: Remote Installation

### ISSUE THREAD

**Issue by cannockwolf** [LAB02-Research/HASS.Agent#265]
Hi does HASS Agent need to be on the same network as the HA Installation to work?

I have set up HASS Agent on my work computer to get window popups but the MQTT server wont connect using https://xxxxxxx.duckdns.org:8123/ in the API settings

**Comment by LAB02-Admin** [comment]
Not necessarily. But you need to be able to reach your mqtt broker as well (which isn't available on the same port as home assistant, usually it's 1883). If you're going to open that to the internet, you're going to need some very strict security and it's overall not recommended.

You could look into using zerotier and the zerotier addon, that way you should be able to connect to your instance from whereever.

**Comment by cannockwolf** [comment]
Thanks 👍🏼
